core/src/main/java/com/sequenceiq/cloudbreak/service/ldapconfig/LdapConfigService.java

```diff
@@ -41,7 +42,8 @@ public LdapConfig create(IdentityUser user, LdapConfig ldapConfig) {
         try {
             return ldapConfigRepository.save(ldapConfig);
         } catch (DataIntegrityViolationException ex) {
-            throw new BadRequestException(getProperSqlErrorMessage(ex));
+            String msg = String.format("Error with resource [%s], error: [%s]", APIResourceType.RDS_CONFIG, getProperSqlErrorMessage(ex));
```

Comment:
Here the LDAP_CONFIG resource type should be used.

Reply:
fixed